agent_server.py

```
@@ -114,6 +114,151 @@ class Modules:
+                text = f"[AGENT TASK ASSIGNED] method={method} | {desc}"
+            elif kind == "completed":
+                text = f"[AGENT TASK COMPLETED] method={method} | {desc}"
+                if detail:
+                    text += f" | result: {detail}"
+            else:
+                text = f"[AGENT TASK FAILED] method={method} | {desc}"
+                if detail:
+                    text += f" | error: {detail}"
+            rec_msgs.append((r["ts"], {"role": "system", "content": text}))
+
+        # 如果原始消息都没有有效时间戳（都是伪时间 0..N），
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)